scripts/package.json

```diff
@@ -7,9 +7,13 @@
         "type-check": "tsc --build"
     },
     "dependencies": {
+        "chalk": "^5.0.0",
+        "fs-extra": "^10.0.1",
+        "minimatch": "^3.0.4",
         "prettier": "2.5.1",
-        "yargs": "17.3.1",
-        "minimatch": "^3.0.4"
+        "prompt-sync": "^4.2.0",
```

Comment:
Why using this poorly maintained npm module https://github.com/heapwolf/prompt-sync ? 

I think we could use just the `process.argv?.[2]` option, and give some error when not provided.

This will allow us to get rid of poorly maintained dependency `prompt-sync`.

Or if CLI is really desired, just for one question, I think using native [readline](https://nodejs.org/api/readline.html) is not that bad.

Reply:
Yea, I will remove it and use just `argv`. I planned to add more options(questions) and but in the end I didn't need that.

Reply:
If you are planning to extend it with more complex questions why not using https://github.com/tj/commander.js which is a very popular library and gives you many options?

Reply:
for now I used just `argv`, better keep it simple now, we can add it later